src/Mod/Sketcher/Gui/TaskSketcherMessages.cpp

```diff
@@ -111,26 +111,26 @@ void TaskSketcherMessages::slotSetUp(const QString& state, const QString& msg, c
 
 void TaskSketcherMessages::on_labelConstrainStatusLink_linkClicked(const QString &str)
 {
-    if( str == QString::fromLatin1("#conflicting"))
+    if (str == QString::fromLatin1("#conflicting"))
         Gui::Application::Instance->commandManager().runCommandByName("Sketcher_SelectConflictingConstraints");
     else
-    if( str == QString::fromLatin1("#redundant"))
+    if (str == QString::fromLatin1("#redundant"))

```

Comment:
why not also add more useful tooltips for these too?
![image](https://user-images.githubusercontent.com/36372335/197846958-fa8cb0b6-8ef3-4e4f-9666-996bde0868b1.png)

Reply:
I think we can't, the "tooltip" there is the link it follows, and has to be that exact text (or at least, cannot be translated).

Reply:
Yes, and this is a problem. As I wrote for a non-native speaker "DoF" is not clear as being an abbreviation nor what it stands for,
But also adding the tooltip does not help as it should when it is not translatable. @adrianinsaval is right, the other "#..." tooltips are also not useful for all users and not translatable.

Unfortunately I don't have a good idea. In my opinion the link should in general be replaced by a button to follow our general design of dialogs. Maybe when doing this, the tooltip can be fixed? @abdullahtahiriyo ?

Reply:
We can replace the link url label by the corresponding button we had in the toolbar. And the number of constraints could be put in the 'Redundant constraints:' label.
So it would be : 'Redudant constraints: 5' label +  SelectRedundantToolButton.
This way we solve all the problems: 
- 'DoF' string meaning
- Tooltip not working
- width of the text will be smaller.

Reply:
Sounds good. Could you make an according PR?